src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 200px;
+  font-style: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+}
```

Comment:
Follow the checklist to add fonts correctly 
<img width="1095" alt="image" src="https://user-images.githubusercontent.com/99403198/216771144-adb60daf-4401-4739-99c3-40c149c6ac77.png">

Reply:
still not fixed